src/notes.js

```diff
@@ -47,10 +78,14 @@ const readNode = async(title) => {
     }else{
         return (undefined)
```

Comment:
Why undefined? If there is no note of that title, then what should be the response?

Reply:
actually i forget to remove this function i updated the code for CRUD operation and to read notes i am using listNotes function... this is some waste code i will remove it